app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/coursedetail/recentactivity/RecentActivityFragment.kt

```diff
@@ -82,13 +82,11 @@ class RecentActivityFragment : Fragment() {
         })
     }
 
-    /**
+   /**
      *   This shows the message
      */
-    fun initialzeToaster() {
-        recentActivityViewModel.showMsg.observe(this, androidx.lifecycle.Observer {
-            val message = it.showMsg
-            context?.showToast(message)
-        })
+    override fun showMessage(message: String) {
+        view?.showSnackbar(message)
+      
     }
```

Comment:
don't remove the initialiseToaster method. 

Just remove             
'context?.showToast(message)'
and instead use 
'view?.showSnackbar(message)'

Reply:
Also, `initializeToaster` will be renamed to `initialiseSnackbar`

Reply:
okay, got it.